codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -71,6 +85,46 @@ public void onStreamRemoved(Http2Stream stream) {
                 }
             }
         });
+
+        supportsCompressionOptions = false;
+    }
+
+    public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate,
+                                            CompressionOptions... compressionOptions) {
+        this(delegate, Arrays.asList(ObjectUtil.checkNotNull(compressionOptions, "CompressionOptions")));
+    }
+
+    public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate,
+                                            Iterable<CompressionOptions> compressionOptionsIterable) {
```

Comment:
`HttpContentCompressor` takes only varargs, but this one also has an overload for `Iterable`. Can we be consistent and either remove this or add `Iterable` overload for `HttpContentCompressor`?

Reply:
`varargs` is good when there are a couple of variables but after a certain limit, it can be painful. `Iterable` can be used with `List` or something equivalent. So I think, supporting both ways is a good idea.

Reply:
I think we should just do varargs for now and come back to this if we need too

Reply:
Alright then, making changes.